### PR TITLE
Release/alpha 0.0.1

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/ProjectService.Business/Commands/Task/EditTaskCommand.cs [hunk 1]
```diff
@@ -76,44 +76,32 @@ private IGetDepartmentResponse GetDepartment(Guid userId, List<string> errors)
...
 
-            bool isDepartmentDirector = false;
-            if (department != null)
-            {
-                 isDepartmentDirector = department.DirectorUserId == requestUserId;
-            }
-
-            if (!isAdmin && !isProjectParticipant && !isDepartmentDirector)
+            if (!_accessValidator.IsAdmin()
+                && !(_userRepository
```
Comment (BelehovEgor) [inline src/ProjectService.Business/Commands/Task/EditTaskCommand.cs:94]: we have to rework this check, need to create a method in the repository, which will try to find one record of project employee. Get all list of project users here is bad, I guess.

File: src/ProjectService.Business/Commands/Task/GetTaskCommand.cs [hunk 2]
```diff
@@ -131,17 +131,17 @@ public OperationResultResponse<TaskResponse> Execute(Guid taskId, bool isFullMod
...
+                userIds.Add(task.AssignedTo.Value);
             }
 
             if (task.ParentTask != null)
             {
                 userIds.Add(task.ParentTask.AuthorId);
             }
 
-            var parentTaskAssignedTo = task.ParentTask?.AssignedTo.GetValueOrDefault();
+            Guid? parentTaskAssignedTo = task.ParentTask?.AssignedTo;
```
Comment (BelehovEgor) [inline src/ProjectService.Business/Commands/Task/GetTaskCommand.cs:144]: we can remove this variable

File: src/ProjectService.Data/Interfaces/ITaskPropertyRepository.cs [hunk 3]
```diff
@@ -13,6 +14,8 @@ public interface ITaskPropertyRepository
 
         bool AreExist(params Guid[] ids);
 
+        bool AreExist(Guid id, TaskPropertyType type);
```
Comment (BelehovEgor) [inline src/ProjectService.Data/Interfaces/ITaskPropertyRepository.cs:17]: Does exist?

Comment (BelehovEgor) [reply src/ProjectService.Data/Interfaces/ITaskPropertyRepository.cs:17]: we need to rename all this (3) methods

File: src/ProjectService.Data/Interfaces/ITaskRepository.cs [hunk 4]
```diff
@@ -24,7 +24,7 @@ public interface ITaskRepository
 
         IEnumerable<DbTask> Find(FindTasksFilter filter, IEnumerable<Guid> projectIds, int skipCount, int takeCount, out int totalCount);
 
-        Guid CreateTask(DbTask item);
+        Guid Create(DbTask item);
 
         bool IsExist(Guid id);
```
Comment (BelehovEgor) [inline src/ProjectService.Data/Interfaces/ITaskRepository.cs:29]: same

File: src/ProjectService.Data/Interfaces/ITaskPropertyRepository.cs [hunk 5]
```diff
@@ -13,6 +14,8 @@ public interface ITaskPropertyRepository
 
         bool AreExist(params Guid[] ids);
```
Comment (BelehovEgor) [inline src/ProjectService.Data/Interfaces/ITaskPropertyRepository.cs:15]: Do exist?

**Review by BelehovEgor — APPROVED** [review]

File: src/ProjectService.Data/Interfaces/ITaskPropertyRepository.cs (hunk 3 above)
Comment (djino97) [reply src/ProjectService.Data/Interfaces/ITaskPropertyRepository.cs:17]: Why?

File: src/ProjectService.Data/TaskRepository.cs [hunk 6]
```diff
@@ -113,13 +111,13 @@ public DbTask Get(Guid taskId, bool isFullModel)
...
-                            .Include(t => t.Status)
-                            .Include(t => t.Project)
-                            .AsSingleQuery()
-                            .AsQueryable();
+            IQueryable<DbTask> dbTasks = _provider.Tasks
+                .Include(t => t.Priority)
+                .Include(t => t.Type)
+                .Include(t => t.Status)
+                .Include(t => t.Project)
+                .AsSingleQuery()
```
Comment (djino97) [inline src/ProjectService.Data/TaskRepository.cs:119]: remove .AsSingleQuery()

**Review by djino97 — APPROVED** [review]

**Pull request merged**